content/post/blackboard_submission_service.md

```diff
@@ -0,0 +1,105 @@
++++
+date = "2015-01-19T19:50:00+10:00"
+draft = true
+title = "Blackboard Extensions and the Submission Service"
+description = "Stepping through the process for adding the required fields to your bb-manifest file."
+keywords = ["blackboard"]
++++
+
+# Blackboard Extensions and the Submission Service
```

Comment:
No need for this heading. The blog theme grabs the heading from the title attribute above.